### PR TITLE
Update to graphcool framework

### LINKED ISSUE #59: Figure out how to upgrade to the new framework of Graphcool
This should greatly improve our collaboration abilities on this project.
https://blog.graph.cool/graphcool-framework-preview-ff42081b1333

### PR DESCRIPTION
**Pull request by wdoug**
This PR closes #59

### CODE REVIEW

**Comment by wdoug** [comment]
I don't know why travis-ci is even running for this project yet. It isn't configured

File: package.json [hunk 1]
```diff
@@ -8,8 +8,7 @@
   },
   "dependencies": {
     "apollo-client": "^1.9.0",
-    "auth0-js": "^8.8.0",
-    "auth0-lock": "^10.19.0",
```
Comment (dviramontes) [inline package.json:12]: @wdoug do we loose the auth0 lock interface ? its cool but we'll have to come up with a login interface of our own ? or maybe we add auth0 lock later?

File: README.md [hunk 2]
```diff
@@ -8,8 +8,22 @@ Node.js application that connects volunteers with volunteers/projects based on t
...
-- [Node.js](https://nodejs.org) - version 8.4 or higher (if you have nvm type `nvm use` in the repo to use the correct version)
-- graph.cool CLI: `npm install -g graphcool`
+
+- [Node.js](https://nodejs.org) - version 8.4 (if you have nvm type `nvm use` in the repo to use the correct version)
+
+Setup:
+1. clone repo
+2. cd into repo directory
+3. `npm install`
+4. Follow [these instructions to set up Auth0](https://github.com/graphcool/templates/tree/master/auth/auth0#3-setup-auth0) for the project.
```
Comment (jksmall0631) [inline README.md]: Change "localhost:8080" to "localhost:3000" for new screenshot of auth0 profile setup

**Comment by nlkluth** [comment]
@wdoug that's my bad. we were working on #57 last time 😨

File: README.md [hunk 3]
```diff
@@ -8,8 +8,22 @@ Node.js application that connects volunteers with volunteers/projects based on t
...
+5. export environment variables:
+- `REACT_APP_AUTH0_CLIENT_ID` as your Auth0 client id
+- `REACT_APP_AUTH0_DOMAIN` and `AUTH0_DOMAIN` as your Auth0 domain
+- `AUTH0_API_IDENTIFIER` as your Auth0 api identifier
+7. install graph.cool CLI: `npm install -g graphcool`
```
Comment (jksmall0631) [inline README.md]: Install in dev dependencies

File: package.json (hunk 1 above)
Comment (dviramontes) [reply package.json:12]: Talked to @wdoug RT / we'll work on a login view later or add this package as needed.

File: README.md [hunk 4]
```diff
@@ -8,8 +8,22 @@ Node.js application that connects volunteers with volunteers/projects based on t
...
-- graph.cool CLI: `npm install -g graphcool`
+
+- [Node.js](https://nodejs.org) - version 8.4 (if you have nvm type `nvm use` in the repo to use the correct version)
+
+Setup:
+1. clone repo
+2. cd into repo directory
+3. `npm install`
+4. Follow [these instructions to set up Auth0](https://github.com/graphcool/templates/tree/master/auth/auth0#3-setup-auth0) for the project.
+5. export environment variables:
```
Comment (jksmall0631) [inline README.md]: Put all env vars in .env.local file in root

**Comment by KeynesYouDigIt** [comment]
Will pull and test and review next monday

**Review by dviramontes — APPROVED** [review]
👍 / looks great! thanks!

**Pull request merged**